content/_layouts/corebaseline.html.haml

```diff
@@ -19,4 +19,4 @@ layout: developersection
     # Surely there's going to be at least one plugin depending on the old .1 by the time the next .1 is available.
     previous_lts_highs = lts.map { |v| v[0...-1] }.uniq[0...-1].map { |l| all_lts.select { |v| v.start_with?(l) }[0] }
 
-= page.content.gsub('PLACEHOLDER_NEWER_LTS_POINT_ONE', next_lts_1).gsub('PLACEHOLDER_RECENT_LTS_POINT_HIGHS', previous_lts_highs[-2...].join(' and ')).gsub('PLACEHOLDER_OLDER_LTS_POINT_HIGHS', previous_lts_highs[0...-2].join(' and ')).gsub('PLACEHOLDER_OLDEST_LTS', oldest_lts).gsub('PLACEHOLDER_OLDEST_WEEKLY', oldest_weekly).gsub('PLACEHOLDER_LATEST_SPLIT', latest_split).gsub('PLACEHOLDER_RECENT_LTS_POINT_HIGH', previous_lts_highs[-1])
+= page.content.gsub('PLACEHOLDER_NEWER_LTS_POINT_ONE', next_lts_1).gsub('PLACEHOLDER_RECENT_LTS_POINT_HIGHS', previous_lts_highs[-2...].join(' and ')).gsub('PLACEHOLDER_OLDEST_WEEKLY', oldest_weekly).gsub('PLACEHOLDER_LATEST_SPLIT', latest_split).gsub('PLACEHOLDER_RECENT_LTS_POINT_HIGH', previous_lts_highs[-1])
```

Comment:
The PLACEHOLDER_OLDEST_LTS is still referenced in the guidance for API plugins.  I believe that reference is correct and should be allowed to continue.  API plugins are different and need a different baseline than other plugins.
```suggestion
= page.content.gsub('PLACEHOLDER_NEWER_LTS_POINT_ONE', next_lts_1).gsub('PLACEHOLDER_RECENT_LTS_POINT_HIGHS', previous_lts_highs[-2...].join(' and ')).gsub('PLACEHOLDER_OLDEST_LTS', oldest_lts).gsub('PLACEHOLDER_OLDEST_WEEKLY', oldest_weekly).gsub('PLACEHOLDER_LATEST_SPLIT', latest_split).gsub('PLACEHOLDER_RECENT_LTS_POINT_HIGH', previous_lts_highs[-1])
```